ml-algorithms/src/main/java/org/opensearch/ml/engine/MLEngine.java

```diff
@@ -34,13 +37,18 @@ public MLEngine(Path opensearchDataFolder) {
         djlModelsCachePath = djlCachePath.resolve("models_cache");
     }
 
-    public String getCIPrebuiltModelConfigPath(String modelName, String version) {
-        return String.format("https://ci.opensearch.org/ci/dbc/models/ml-models/%s/%s/config.json", modelName, version, Locale.ROOT);
+    public String getPrebuiltModelConfigPath(String modelName, String version, MLModelFormat modelFormat) {
+        String format = modelFormat.name().toLowerCase(Locale.ROOT);
+        return String.format("%s/%s/%s/%s/config.json", MODEL_REPO, modelName, version, format, Locale.ROOT);
```

Comment:
Should we check if the file is actually there? May be raise an exception if the model config path is not valid?

Reply:
The download function will throw exception if file doesn't exist.